pkg/forwarder/interceptor/interceptor.go

```diff
@@ -107,6 +124,51 @@ func (i *interceptor) CreateContainer(ctx context.Context, req *pb.CreateContain
 	return res, err
 }
 
+func isTargetPath(path, targetPath string) bool {
+	return targetPath != "" && targetPath == path
+}
+
+func waitForDeviceMounted(path string) error {
+	timer := time.NewTimer(volumeTimeout)
+
+	for {
+		isMounted := isMounted(execFindmnt, path)
```

Comment:
How about utilizing this existing library?

https://pkg.go.dev/github.com/moby/sys/mountinfo

Reply:
Great, I'll have a try.